x-pack/solutions/observability/plugins/infra/public/pages/metrics/inventory_view/components/waffle/node_context_menu.tsx

```diff
@@ -140,8 +143,6 @@ export const NodeContextMenu = withEuiTheme(
         values: { inventoryName: inventoryModel.singularDisplayName },
       }),
       href: nodeDetailMenuItemLinkProps.href,
-      onClick: nodeDetailMenuItemLinkProps.onClick,
```

Comment:
Are you sure that removing `onClick` will trigger an issue with a full-page reload? What if we simply add `role: "link"` to fix that without any risks

Reply:
I checked, and it was not triggering a full page reload

https://github.com/user-attachments/assets/68b736eb-05ec-4412-95f2-3fce38465ac6